### PR TITLE
Fix minor localization issues in Spanish:

### PR DESCRIPTION
**Pull request by elenatorro**
Minor things I've seen while using the app and then reviewing some of the texts:

- Use 'pack de inicio' instead of 'paquete de inicio'
- Use 'Inténtalo' instead of 'Intenta'
- Fix 'Por favor'
- Do not use 'una' as article for 'posts'
- Use 'Eliminado' instead of 'Removido'

### CODE REVIEW

**Comment by elenatorro** [comment]
Hi @madkoding! I'm pinging you since I saw your changes on this pull request https://github.com/bluesky-social/social-app/pull/5842 I'm not sure who I should ping to review this. Is there any guide for language contribution? Is it necessary to create an issue for these minor changes?

Thanks!

**Comment by surfdude29** [comment]
Pinging @oledfish and @Kloen for input as the other reviewers on #5842 :)

**Comment by surfdude29** [comment]
@elenatorro One tip for the future from a fellow translator: It's not necessary to translate any strings where the line starts with `#~`. These are obsolete strings that are no longer used in the app, they just haven't been removed from the message catalog yet.

Some translators like to remove them to keep things tidy, but it's not necessary to do this as they'll just be ignored if they're left in.

**Comment by elenatorro** [comment]
@surfdude29 thanks for the tip! I had no idea about that, I can change it back 👍🏽

File: src/locale/locales/es/messages.po
Comment (ecreeth) [inline src/locale/locales/es/messages.po:136]: Why should we use `pack` instead of `paquete`? we're trying to localize the app :) sounds better for me but isn't correct for most users.

Maybe we should consider another text?

File: src/locale/locales/es/messages.po
Comment (ecreeth) [inline src/locale/locales/es/messages.po:653]: `denuevo` -> `de nuevo`

**Comment by oledfish** [comment]
I personally agree with @ecreeth in "paquete" over "pack". I think it's best to only defer to loanwords when there's no good equivalent in Spanish.

File: src/locale/locales/es/messages.po
Comment (elenatorro) [reply src/locale/locales/es/messages.po:136]: Ok! I have no strong opinion here, maybe I'm too biased as I use "pack" in Spanish way more than "paquete" 😂 I'll change it back

**Comment by oledfish** [comment]
@elenatorro, the changes look good but I cannot approve them for merging myself, a maintainer will have to do that

File: src/locale/locales/es/messages.po
Comment (ecreeth) [inline src/locale/locales/es/messages.po:6536]: `Compartir imagen` -> `Compartir imagen externamente`

File: src/locale/locales/es/messages.po
Comment (ecreeth) [inline src/locale/locales/es/messages.po:653]: IMHO 😄  `inténtelo` -> `intente`

File: src/locale/locales/es/messages.po
Comment (gonzaloriestra) [inline src/locale/locales/es/messages.po:657]:
```suggestion
msgstr "Se produjo un error al cargar el video. Por favor, inténtelo de nuevo."
```

File: src/locale/locales/es/messages.po
Comment (gonzaloriestra) [inline src/locale/locales/es/messages.po:674]:
```suggestion
msgstr "Se produjo un error al seleccionar el video"
```

File: src/locale/locales/es/messages.po
Comment (gonzaloriestra) [inline src/locale/locales/es/messages.po:683]:
```suggestion
msgstr "Se produjo un error al intentar seguir a todos"
```

File: src/locale/locales/es/messages.po
Comment (gonzaloriestra) [inline src/locale/locales/es/messages.po:1990]:
```suggestion
msgstr "¿Borrar esta publicación?"
```

File: src/locale/locales/es/messages.po
Comment (gonzaloriestra) [inline src/locale/locales/es/messages.po:5462]:
```suggestion
#~ msgstr "Citar publicación"
```

File: src/locale/locales/es/messages.po
Comment (gonzaloriestra) [inline src/locale/locales/es/messages.po:5467]:
```suggestion
#~ msgstr "Citar publicación"
```

File: src/locale/locales/es/messages.po
Comment (gonzaloriestra) [inline src/locale/locales/es/messages.po:6540]:
```suggestion
#~ msgstr "Compartir imagen en una publicación"
```

File: src/locale/locales/es/messages.po
Comment (gonzaloriestra) [inline src/locale/locales/es/messages.po:6720]:
```suggestion
#~ msgstr "Mostrar las publicaciones de {0} en tu feed"
```

**Review by gonzaloriestra — COMMENTED** [review]
Holi! 👋 

Nice work! May I suggest including [this commit](https://github.com/bluesky-social/social-app/pull/6395/commits/9d3016472c589ac7f3e8dc02bdc435a7710b0793) here?

File: src/locale/locales/es/messages.po
Comment (gonzaloriestra) [reply src/locale/locales/es/messages.po:653]: I vote for `inténtalo` 😄. Anyway, it should be `inténtalo` or `intenta` (_de tú, no usted_).

**Review by pfrazee — APPROVED** [review]
Thank you everybody! I'm going to go ahead and merge so these updates can make their way in. Please feel free to keep tuning the translations in future PRs.

**Comment by hogaza** [comment]
Hi elenatorro and others! I joined bluesky a while ago and I'd be happy to contribute to the localization efforts in Spanish. I'm a professional translator, although it's been a while since I have been involved in any software localization project, so I'm excited to learn the in and outs, git, etc from more experienced peers. 

I can also contribute to strategic language planning (tone and voice, style guide, consistency, etc.), specially if the project moves to Crowdin (#2940).

I would love to connect and collaborate with the rest of the team localizing into Spanish!

File: src/locale/locales/es/messages.po
Comment (elenatorro) [reply src/locale/locales/es/messages.po:6720]: I just saw these! maybe we you include them in another PR? 🙏🏽

**Comment by elenatorro** [comment]
Thanks @pfrazee! I was late to apply some suggestions left in this PR, but I they can be fixed in following PRs, happy to help if needed.

**Pull request merged**
